src/base.ml

```diff
@@ -19,8 +29,6 @@ type tick_func = int -> Screen.t -> Framebuffer.t -> input_state -> Framebuffer.
 
 type bitmap_t = (int32, Bigarray.int32_elt, Bigarray.c_layout) Bigarray.Array1.t
 
-type functional_tick_func = int -> Screen.t -> KeyCodeSet.t -> Primitives.t list
```

Comment:
In the merge it's deleted code that should be removed - can you restore this back?